sdk/sphere/Azure.ResourceManager.Sphere/src/autorest.md

```diff
@@ -7,7 +7,8 @@ azure-arm: true
 csharp: true
 library-name: Sphere
 namespace: Azure.ResourceManager.Sphere
-require: https://github.com/Azure/azure-rest-api-specs/blob/a89f3906ba60257ae28a2eed756a1ee4ca72ed51/specification/sphere/resource-manager/readme.md
+require: https://github.com/Azure/azure-rest-api-specs/blob/12457329527160696e47221e2078929834e9f60c/specification/sphere/resource-manager/readme.md
```

Comment:
Need update the default tag in the readme file to `package-2024-04-01`

Reply:
okay I have opened a PR for this: https://github.com/Azure/azure-rest-api-specs/pull/28178